lib/aggregate_all_epitopes.py

```diff
@@ -0,0 +1,206 @@
+import pandas as pd
+import numpy as np
+
+class AggregateAllEpitopes:
+    def __init__(self, input_file, output_file, file_type='pVACseq'):
+        self.input_file = input_file
+        self.output_file = output_file
+        self.file_type = file_type
+
+    #assign mutations to a "Classification" based on their favorability
+    def get_tier(self, mutation, vaf_clonal):
+        anchor_residue_pass = True
+        anchors = [1, 2, len(mutation["MT Epitope Seq"])-1, len(mutation["MT Epitope Seq"])]
+        if mutation["Mutation Position"] in anchors and mutation["Median WT Score"] < 1000:
```

Comment:
at some point, we're probably going to want to parameterize a lot of these (scores, expression levels, etc), but I think hardcoded is fine for this first version.